### PR TITLE
Add earthfile and manual ghcr container creation

### PR DESCRIPTION
**Pull request by cosmosgenius**

### CODE REVIEW

File: .github/workflows/manual_ghcr.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Container Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.APPKNOX_GITHUB_CONTAINER_TOKEN }} #encrypted secret var
+          registry: ${{ github.event.inputs.registry }}
+      - name: Download latest earthly
+        run: "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/download/v0.6.2/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
```
Comment (Praseetha-KR) [inline .github/workflows/manual_ghcr.yaml:23]: @cosmosgenius can we use latest version `https://github.com/earthly/earthly/releases/latest/downloads/earthly-linux-amd64` ?

File: .github/workflows/manual_ghcr.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Container Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.APPKNOX_GITHUB_CONTAINER_TOKEN }} #encrypted secret var
+          registry: ${{ github.event.inputs.registry }}
```
Comment (Praseetha-KR) [inline .github/workflows/manual_ghcr.yaml:21]: `github.event.inputs.registry` this should not be manual input right?

**Review by Praseetha-KR — CHANGES_REQUESTED** [review]

File: .github/workflows/manual_ghcr.yaml (hunk 1 above)
Comment (cosmosgenius) [reply .github/workflows/manual_ghcr.yaml:23]: this was copied from earth docs https://docs.earthly.dev/ci-integration/vendor-specific-guides/gh-actions-integration

Comment (cosmosgenius) [reply .github/workflows/manual_ghcr.yaml:23]: lets keep it that way for time being

File: .github/workflows/manual_ghcr.yaml (hunk 2 above)
Comment (cosmosgenius) [reply .github/workflows/manual_ghcr.yaml:21]: ya this is a mistake ghrc.io is hardcoded now

**Review by Praseetha-KR — APPROVED** [review]

**Pull request merged**